tests/integration/test_project.py

```diff
@@ -64,3 +65,25 @@ def test_extend_reservations(project):
     assert project.extend_reservations("ReviewQueue") == 0
     with pytest.raises(InvalidQueryError):
         project.extend_reservations("InvalidQueueType")
+
+
+def test_attach_instructions(client, project):
+    with pytest.raises(ValueError):
+        project.attach_instructions('/tmp/instructions.txt')
+
+    editor = list(
+        client.get_labeling_frontends(
+            where=LabelingFrontend.name == "editor"))[0]
+    empty_ontology = {"tools": [], "classifications": []}
+    project.setup(editor, empty_ontology)
+
+    with open('/tmp/instructions.txt', 'w') as file:
+        file.write("some instructions...")
+
+    project.attach_instructions('/tmp/instructions.txt')
+    assert json.loads(
+        list(project.labeling_frontend_options())
+        [-1].customization_options).get('projectInstructions') is not None
```

Comment:
would it make sense to verify that this is the same test as "some instructions ..."

Reply:
There is no way to verify because it is uploaded to google cloud store and the client doesn't have permissions.